build/Nuget/Sdkproject.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Maybe you can add a comment here pointing at `https://github.com/NuGet/Home/wiki/Adding-nuget-pack-as-a-msbuild-target#packing-using-a-nuspec` to explain why we have this project?